pom.xml

```diff
@@ -1065,7 +1065,7 @@
             <dependency>
                 <groupId>org.projectlombok</groupId>
                 <artifactId>lombok</artifactId>
-                <version>1.18.20</version>
+                <version>1.18.30</version>
```

Comment:
lombok was added for a very special case (long story...), and in theory should be removed when i have time (was coming from an external PR)